### DISCUSSION TITLE
Apache proxy how to ?

### DISCUSSION THREAD

**Discussion by tompointexe** [Impostor/Impostor#328] [Q&A]
Hi, Have anyone proxyed there impostor server instance with apache ?? if so i'm interested so i could do it too :)

**Comment by dimaguy** [comment]
Theres no point in doing apache reverse proxy.
Just run impostor in the place where apache would be as it uses nearly no resources

**Comment by tompointexe** [comment]
for me there is a point as it's my main and most powerfull server, i use it to host Imposor, Crewlink, and my website ... i cant run apache and impostor both at the same time on port 80 and 433 🤔 (And crewlink is proxyed by the apache server) (if impostor is just a WS with the game i think i will be able to do it myself 😅 )

**Comment by dimaguy** [comment]
You do realize that impostor uses udp not tcp and can actually run at the same time on those ports right?

**Comment by tompointexe** [comment]
👀 i'l try it :) thanks for the help

**Comment by tompointexe** [comment]
![image](https://user-images.githubusercontent.com/29174751/106941832-b1d56280-6723-11eb-9212-fc4203372133.png)
Well it has well started untill ... :(

**Comment by dimaguy** [comment]
Surprise you need to listen at port 80 too

**Comment by tompointexe** [comment]
That's what i did on the 2nd try ... :eyes:

**Comment by dimaguy** [comment]
Do you have anything udp listening in port 80?

**Comment by tompointexe** [comment]
Euh ... apache isnt udp ? if so i have apache :joy:
else i dont think so

**Comment by dimaguy** [comment]
No apache is tcp.

**Comment by Mm2PL** [comment]
Try replacing port 80 in your config with the default, 22023. Also could the other listening thing be another instance of the server?

**Comment by tompointexe** [comment]
yes with the default it works (with my isp public ip not with cloudflare) but on port 80 it does not, (cloudflare is the thing that's makes me proxy everything thru apache (but it add "security") )

**Comment by dimaguy** [comment]
Cloudflare wouldnt let you do it proxied anyways.
Use linux networking tools to figure out whats bogging port 80 udp and terminate the process.
It is most likely a zombie impostor server process running in there

**Comment by tompointexe** [comment]
udp        0      0 0.0.0.0:80              0.0.0.0:*                           -
udp6       0      0 [::]:80                 [::]:*                               -

A non existant program :eyes: i dont actualy like that ... same on UDPV6

**Comment by dimaguy** [comment]
What tool did you use?
Also rebooting might be a good option

**Comment by tompointexe** [comment]
i used netstat, ss & lsof ... impossible to identify the udp 80  program

**Comment by dimaguy** [comment]
> 
> 
> i used netstat, ss & lsof ... impossible to identify the udp 80 program

Either identify it or move onto 443

**Comment by js6pak** [comment]
Among Us still has 22023 hard coded in few places, its better to just use it.
Pretty sure `Permission denied` means that system denied your access to the port, can you try running impostor with `sudo`?
If you really want, you can proxy udp with nginx/apache, just google it, but its not really needed.
And I'm pretty sure that cloudflare doesn't proxy udp.

**Reply by tompointexe** [reply]
Hi, i'l think about that ... maybe doing an "srv record" could be enough so i dont have to give my public ip everytime (Yeah just thinking about the srv record now 👀 LUL

**Comment by AeonLucid** [comment]
No, this is not an http server.

**Reply by tompointexe** [reply]
no like a srv record like a minecraft server ... i'l try it on my side
